### PR TITLE
Carousel effect

### LINKED ISSUE #7: Create a carousel effect with some test images
Like what the lektor page provides, 

You can use some images from here, https://github.com/spyder-ide/website/tree/master/img_src

### PR DESCRIPTION
**Pull request by rlaverde**
Add a demo carousel effect to home page using [slick-carousel](https://kenwheeler.github.io/slick/)
Also fixes an error when loading fonts (missing publicPath in webpack config)
Fixes #7

### CODE REVIEW

File: webpack/webpack.config.js [hunk 1]
```diff
@@ -28,11 +29,11 @@ module.exports = {
...
           'style-loader', 'css-loader!postcss-loader') },
       { test: /\.(png|jpe?g|gif)$/,
-        loader: 'file?name=images/[name].[ext]' },
+        loader: 'file?name=/images/[name].[ext]' },
       { test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
-        loader: "url-loader?limit=10000&mimetype=application/font-woff&name=fonts/[name].[ext]" },
+        loader: "url-loader?limit=10000&mimetype=application/font-woff&name=/fonts/[name].[ext]" },
       { test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
-        loader: "file-loader?name=fonts/[name].[ext]" }
+        loader: "file-loader?name=/fonts/[name].[ext]" }
```
Comment (goanpeca) [inline webpack/webpack.config.js:36]: the missing '/' was the error?

**Comment by goanpeca** [comment]
@rlaverde is this ready?

File: webpack/webpack.config.js (hunk 1 above)
Comment (rlaverde) [reply webpack/webpack.config.js:36]: The error was this https://github.com/spyder-ide/spyder-website/pull/28/files#diff-28b51d16260380d9fae7857a4ba6dbeaR13
but after adding that, It was necessary to add '/' to the names

Comment (goanpeca) [reply webpack/webpack.config.js:36]: Got it

**Pull request merged**